crane-cloud/backend
# Upgrade Backend packages

# Description

- Upgrade backend packages

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID

- https://trello.com/c/EJOYaVZO

## How Can This Be Tested?

- Run the upgraded packages against the existing CI pipeline to ensure nothing breaks


## Checklist:
